release-scripts/checkIfPackageShouldBePublished.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const execSync = require("child_process").execSync;
+
+function checkVersion(packageName, version) {
+	return execSync(`npm view ${packageName}@${version}`).toString().trim();
+}
+
+const path = require("path");
+const libPath = path.join(__dirname, '..', process.argv[2]);
+
+const packageName = require(`${libPath}/package.json`).name;
+const currentVersion = require(`${libPath}/package.json`).version;
```

Comment:
Would something like this work, instead of importing twice?

```suggestion
const {name, version} = require(`${libPath}/package.json`);
```